admin/ownership_grants/V1.6.0__proxy_admin_role_dev.sql

```diff
@@ -0,0 +1,21 @@
+-- Grant the proxy admin database role ownership and usage
+-- of the `*ALL_ADMIN` database roles.
+GRANT OWNERSHIP
+    ON DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_ALL_ADMIN
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.ALL_ADMIN
+    COPY CURRENT GRANTS;
+GRANT OWNERSHIP
+    ON DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_RAW_ALL_ADMIN
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.ALL_ADMIN
+    COPY CURRENT GRANTS;
+GRANT OWNERSHIP
+    ON DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SCHEMACHANGE_ALL_ADMIN
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.ALL_ADMIN
+    COPY CURRENT GRANTS;
+
+GRANT DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_ALL_ADMIN
+    TO ROLE SYNAPSE_DATA_WAREHOUSE_DEV.ALL_ADMIN; --noqa: JJ01,PRS,TMP
```

Comment:
nit: No need to have this comment here `--noqa: JJ01,PRS,TMP`, the same applies for the other lines, and V1.6.1